selenium/testcontainers/selenium/__init__.py

```diff
@@ -24,7 +25,7 @@
 }
 
 
-def get_image_name(capabilities: str) -> str:
+def get_image_name(capabilities: Dict[str, Any]) -> str:
```

Comment:
Isn't this a breaking change in the API?

Reply:
it is, but the api definition was incorrect before - it is indexed in the old code like a dict anyways to determine which image to use (ff vs chrome)

Reply:
resolved this, copied to new discussion with whole method quoted